.changeset/nice-ties-sniff.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
+'@sap-ux/preview-middleware': patch
+---
+
+CPE - handling missing entity set in semantic date range qa change
```

Comment:
Changelog should describe what was fixed not how or why. The technical details can be seen in the commit or PR details.
```suggestion
Fixed "Enable/Disable Semantic Date Range in Filter Bar" quick action in SAP Fiori Elements for OData V2 applications when using UI5 version lower than 1.126.
```